### DISCUSSION TITLE
QuarkusCxfProcessor: SEI ClassNotFoundException handling

### DISCUSSION THREAD

**Discussion by famod** [quarkiverse/quarkus-cxf#259] [General]
Hi @dufoli @shumonsharif

can you tell me why the following exception is only logged?
https://github.com/quarkiverse/quarkus-cxf/blob/4ffb75e9f4d3b89e85d1f52e83b6abac9a46fd14/deployment/src/main/java/io/quarkiverse/cxf/deployment/QuarkusCxfProcessor.java#L252-L254

AFAICS this is rather severe and therefore the processor should abort with an exception, no?

**Comment by famod** [comment]
I decided to rethrow this execption in #262

**Reply by shumonsharif** [reply]
Good thing you did @famod ... I don't recall why we were just logging it ... but we had to resolve several class loader issues at some point in the past, and may just have been a remnant of trying to debug the issue back then.

**Reply by famod** [reply]
Sounds plausible! 👍

**Reply by dufoli** [reply]
at the beginning, I avoid all exception and catch them in log to fix all bugs.
